src/style.css

```diff
@@ -1 +1,73 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(/src/fonts/Avenir-Book.ttf) format('opentype') tech(color-COLRv1);
+}
+
+body {
+  margin: 0;
+  padding: 20px;
+  font-weight: 300;
+}
+
+.lab-first img {
+  width: 19px;
+  height: 19px;
+  margin-left: 26px;
+  margin-right: 17px;
+}
+
+.lab-second img {
+  width: 11px;
+  height: 11px;
+  margin-left: 13px;
```

Comment:
Consider using only margin-left or margin-right to avoid potential margin collapse issues. The checklist suggests being consistent with margins.